docs/pages/guides/upgrading/1.0.mdx

```diff
@@ -0,0 +1,169 @@
+
+const client = createClient({
+  /* ... other options ... */
+  throttle: 16,
+});
+```
+
+### Improvements to the Room class [#room]
+
+We added new methods to the `Room` class that you can use to obtain the storage
+status of a room, subscribe to storage status changes, or reconnect to a room
+whenever necessary. You can also use the
+[Liveblocks DevTools extension](/devtools) to visualize real-time changes to
+storage while implementing these changes.
+
+#### Room.getStorageStatus [#getStorageStatus]
+
+The different storage statuses are:
+
+- `not-loaded`: Initial state when entering the room.
+- `loading`: Once the storage has been requested via room.getStorage().
+- `synchronizing`: When some local updates have not been acknowledged by
+  Liveblocks servers.
+- `synchronized`: Storage is in sync with Liveblocks servers.
+
+#### Room.subscribe [#subscribe]
+
+Use `Room.subscribe` to subscribe to storage status changes. This method returns
+an unsubscribe function.
+
+```ts
+room.subscribe("storage-status", (status) => {
+  // Implement your logic here
+  switch (status) {
+    case "not-loaded":
+      break;
+    case "loading":
+      break;
+    case "synchronizing":
+      break;
+    case "synchronized":
+      break;
+    default:
+      break;
+  }
+});
+```
+
+#### Room.reconnect [#reconnect]
+
+You can use `room.reconnect()` to close the room connection and try to
+[reconnect](/docs/api-reference/liveblocks-client#Room.reconnect).
+
+## Changes in @liveblocks/node [#node]
+
+### Update the authorize option [#authorize]
+
+We have updated the `authorize` method to make `userId` mandatory. This change
+ties into our new [pricing model](https://liveblocks.io/pricing), which is based
+on Monthly Active Users (MAU) instead of connections. We use `userId` to track
+MAU associated with a Liveblocks account.
+
+If you do not use the `authorize` function, a cookie will be set on the client
+to track the user. This cookie will be used to track MAU associated with a
+Liveblocks account and will expire after 30 days.
+
+### Webhooks enhancements [#webhooks]
+
+We have added two new events to our Webhooks functionality: `RoomCreatedEvent`
+and `RoomDeletedEvent`. We have also added a `WebhookHandler` class to make it
+easier to verify event requests from Liveblock’s Webhooks functionality. It also
+provides fully typed `WebhookEvents`.
+
+```ts
+import { WebhookHandler } from "@liveblocks/node";
+
+const webhookHandler = new WebhookHandler(process.env.SECRET);
+
+const event = webhookHandler.verifyRequest({
+  headers: req.headers,
+  rawBody: req.body,
+});
+```
+
+Verification of events is critical to ensure that the events are coming from
+Liveblocks and not from a malicious source. Check out our
+[Webhooks guide](https://liveblocks.io/docs/guides/webhooks) for more details.
+
+## Changes for all Liveblocks packages [#all]
+
+When upgrading to 1.0, the primary breaking change to consider is updating the
```

Comment:
Why is this part under `Changes for all Liveblocks packages`?
The change is only for `liveblocks/node`.